test/orders.e2e-spec.ts

```diff
@@ -0,0 +1,109 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './../src/app.module';
+import { SeederService } from '../src/shared/service/seederService.service';
+
+describe('UsersController E2E Test', () => {
+  let app: INestApplication;
+
+  const loginDto = {
+    username: 'vitusbalazs',
+    password: 'password',
+  };
+  const customerId = '33c5965f-e370-43af-8d9a-c877463f31b1';
+  const firstProductId = 'cf3bb5dc-8439-45a5-8e35-823bdb19d1ee';
+  const secondProductId = 'a03f8d3c-c3ee-41a3-b381-63e9728826e2';
+  const locationId = '485229c8-732e-4b0b-bdea-19c249a70b70';
```

Comment:
You could pass these IDs as parameters to the seeding methods to be sure that the created items have these IDs. Another developer might update the seeding method, remove an `INSERT` or change an ID there without knowing the ID value is actually important.

The tests are dependent on these IDs and we should see this from the start via the parameters.

Reply:
done